### ISSUE TITLE
false positives regarding GROUP BY

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2043]
see https://github.com/abapinho/abapTimeMachine/pull/82

```
./src/zcl_timem_parts.clas.abap[90, 7]                 - <t_group> not found, findTop (check_syntax) [E]
./src/zcl_timem_parts.clas.abap[99, 7]                 - <t_group> not found, findTop (check_syntax) [E]
./src/zcl_timem_parts.clas.abap[150, 7]                - Loop, not a table type (check_syntax) [E]
./src/zcl_timem_parts.clas.abap[159, 7]                - Not a structure, FieldChain (check_syntax) [E]
./src/zcl_timem_parts.clas.abap[202, 11]               - NewObject, todo, infer type (check_syntax) [E]
```

**Comment by larshp** [comment]
#2042
